validator/efotrait.py

```diff
@@ -1,17 +1,17 @@
-import requests
+from validator.request.connector import Connector, ConnectorException
```

Comment:
I might have missed something but shouldn't it be `DefaultConnector` there instead of the abstract `Connector` ?
The abstract `request` method won't be working with the default connector `Connector`

Reply:
See comment bellow